### PR TITLE
Bai_Tap_C#

### PR DESCRIPTION
**Pull request by ledaihai**

### CODE REVIEW

File: exc/canBo.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+using System.Net;
+using System.Reflection;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+
+    public class Officer
+    {
+        
+    public string? name { get; set; }
```
Comment (orienthoanguyen) [inline exc/canBo.cs:13]: Please make all the first letters of public variables and methods uppercase

File: exc/nhanVien.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+               role = role;
+            }
+            public override void outPut()
+            {
+                base.outPut();
+                Console.WriteLine(" role: " + role);
+            }
+
+
```
Comment (orienthoanguyen) [inline exc/nhanVien.cs:21]: Remember to clear or format your code before pushing

File: exc3/KhoiB.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace exc3
+{
+    internal class KhoiB : ThiSinh
+    {
+        public const string? TOAN = "TOAN";
```
Comment (orienthoanguyen) [inline exc3/KhoiB.cs:11]: why do you set this as nullable?

Comment (ledaihai) [reply exc3/KhoiB.cs:11]: Sorry about that , this my bad , i will fix it now .

File: exc4/Nguoi.cs [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace exc4
+{
+    internal class Nguoi
+    {
+        
+    public string? Ten { get; set; }
```
Comment (orienthoanguyen) [inline exc4/Nguoi.cs:12]: for string we don't need nullable. Please update for all exc

**Pull request merged**